mypy/checker.py

```diff
@@ -3726,32 +3726,7 @@ def find_isinstance_check(self, node: Expression
             elif refers_to_fullname(node.callee, 'builtins.issubclass'):
                 if len(node.args) != 2:  # the error will be reported elsewhere
                     return {}, {}
-                expr = node.args[0]
-                if literal(expr) == LITERAL_TYPE:
```

Comment:
I would keep the above two lines in this function. To make the refactoring more 1-to-1.